CMakeLists.txt

```diff
@@ -166,6 +190,15 @@ catkin_package(
 #   PATTERN ".svn" EXCLUDE
 # )
 
+# Install everything in the launch/ directory.
+# This CMake macro can also be used to install data files, etc.
+# See http://wiki.ros.org/catkin/CMakeLists.txt#Installing_roslaunch_Files_or_Other_Resources
+# - gordonb 2017-03-15
+install(DIRECTORY launch/
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+  PATTERN ".svn" EXCLUDE
```

Comment:
Agree, just leave the check for .svn.   And forget about my suggestion for .git.